### PR TITLE
Use neatened SCP interface

### PR DESCRIPTION
**Pull request by mundya**
(This PR based on #91)

Since project-rig/rig#196 `SCPPacket` has had default arguments that
indicate a packet being sent into a SpiNNaker machine. This commit uses
those default arguments to clean up the creation of SCP Packets when
using them to transmit Node data into SpiNNaker.

### CODE REVIEW

File: nengo_spinnaker/node_io/ethernet.py [hunk 1]
```diff
@@ -121,18 +121,10 @@ def set_node_output(self, node, value):
...
-                src_x=0, src_y=0,
-                cmd_rc=0, seq=0, arg1=0, arg2=0, arg3=0,
-                data=packet_data
-            )
-            self.out_socket.sendto(packet.bytestring,
-                                   (self._hostname, 17893))
+            data = bytes(tp.np_to_fix(c_value).data)
+            packet = SCPPacket(dest_port=1, dest_cpu=p, dest_x=x, dest_y=y,
+                               cmd_rc=0, arg1=0, arg2=0, arg3=0, data=data)
+            self.out_socket.sendto(packet.bytestring, (self._hostname, 17893))
```
Comment (mossblaser) [inline nengo_spinnaker/node_io/ethernet.py]: `rig.machine_control.consts.SCP_PORT`?

**Comment by mossblaser** [comment]
Aside from nit-picking, LGTM! All three of my least-favourite things: off by one errors and programmers who count from 1...

**Comment by mundya** [comment]
Having `ticks` count from one is mega frustrating.

File: nengo_spinnaker/node_io/ethernet.py (hunk 1 above)
Comment (mundya) [inline nengo_spinnaker/node_io/ethernet.py]: Done.

**Comment by mossblaser** [comment]
Merge at will ;) (and a +3.9% coverage improvement to boot ;) hooray for improving test coverage by removing code :P)

**Comment by mundya** [comment]
Cheers :)

**Pull request merged**